nerdlandbot/helpers/constants.py

```diff
@@ -47,3 +47,6 @@
 DAD_JOKE_URL = 'https://icanhazdadjoke.com'
 EIGHT_BALL_URL = 'https://evening-castle-06130.herokuapp.com/eightball'
 
+#Nerdland server
+NERDLAND_SERVER_ID = 712743925114339328 #bot test server for testing purposes
```

Comment:
Would recommend this just be named DISCORD_SERVER_ID if we intend to keep this repository forkable / clonable for others. It's a more generic name that still serves the same purpose